apps/designer/src/global-stores/store.ts

```diff
@@ -1,16 +1,40 @@
+import { Vector3 } from "three";
```

Comment:
Why is this in a special folder and not in [globalStore](https://github.com/TechnocultureResearch/VVplus/tree/a4390c1dc9fb8be05c8e20abdcf950d766798de1/apps/designer/src/globalStore)?

Reply:
I just made changes to the file which already had the camera props in it. This file structure was made by someone else.